static/src/javascripts/projects/common/views/experiments/quick-survey.html

```diff
@@ -0,0 +1,92 @@
+<style>
```

Comment:
These [JS templates get bundled in with the JS anyway](https://assets.guim.co.uk/javascripts/bootstraps/enhanced/181c49fefa2c68adf4c9a2717551e64e/main.js) (search for `<iframe` for example), so you're not saving anything by putting the CSS in the template - you might as well have a seperate CSS file and make use of SCSS, variables, linting etc